src/distributions/mod.rs

```diff
@@ -95,7 +95,7 @@
 //!   - [`ChiSquared`] distribution
 //!   - [`StudentT`] distribution
 //!   - [`FisherF`] distribution
-//!
+//! - Dirichlet distribution
```

Comment:
The ``[`Dirichlet`]`` link is missing.

Reply:
It's been added now, but `Dirichlet` seems to be mentioned twice in a row